### DISCUSSION TITLE
Sonar frequency slider

### DISCUSSION THREAD

**Discussion by Freelor** [FakeFishGames/Barotrauma#10279] [Ideas and suggestions]
I noticed the Sonar seems to be quite fast and would love the ability to change the sonar ping frequency for stealth, slower movement. I usually end up flipping the sonar ping on and off at a slower interval but it would be nice to be able to choose to slow or speed up the frequency with benefits and drawbacks included in each.

**Comment by mygamingaccount** [comment]
Frequency could be higher when the range is lower.

Otherwise you'd transmit a second ping before the reflections of the first one returned.

If you want to stay stealthy however, just reduce the range. That will also reduce the amplitude of the ping, and thus the range at which the xenos are attracted.
